conf/defaults.yaml

```diff
@@ -140,7 +140,8 @@ supervisor.memory.capacity.mb: 3072.0
 supervisor.cpu.capacity: 400.0
 
 ### worker.* configs are for task workers
-worker.childopts: "-Xmx768m"
+worker.heap.memory.mb: 768
+worker.childopts: "-Xmx%HEAP-MEM%m"
```

Comment:
@revans2 and I think it would be better practice if we put the HEAP-MEM in the substitute-childopts function together with WORKER-ID, TOPOLOGY-ID, WORKER-PORT.